mmv1/templates/terraform/examples/node_group_maintenance_interval.tf.erb

```diff
@@ -0,0 +1,18 @@
+resource "google_compute_node_template" "soletenant-tmpl" {
+  provider  = google-beta
+  name      = "<%= ctx[:vars]['template_name'] %>"
+  region    = "us-central1"
+  node_type = "c2-node-60-240"
```

Comment:
Got a quota error in our test env:
```
Error waiting for Creating NodeGroup: Quota 'C2_CPUS' exceeded.  Limit: 24.0 in region us-central1.
```
Can we use a smaller node type?

Reply:
In terms of CPU and RAM, this is the lower we can go, but if the problem is that it's C2 and not N2, we could go with `n2-node-80-640`. Otherwise, I would just bump up the quota - if you let me know the project I can bump it up for you in that region.